.github/actions/nm-build-docker/action.yml

```diff
@@ -9,7 +9,7 @@ inputs:
     description: "additional tag for the docker image"
     type: string
```

Comment:
nit: "type" isn't really supported for inputs of "actions"

Reply:
Thanks, good catch, updated.